### ISSUE TITLE
Enjoy using vue-aplayer in your project? Let me know!

### ISSUE THREAD

**Issue by SevenOutman** [SevenOutman/vue-aplayer#26]
Thanks for loving it! Sharing your project and giving feedbacks are welcome.

**Comment by Shadomax** [comment]
Yes. Thank you very much for these awesome vue js plugin. I would like to know how to access event from any component because i want to be able to watch the audio and end it pro-grammatically at a certain time. 
Any clues?

**Comment by SevenOutman** [comment]
@Shadomax Since `v1.4.0` Vue-APlayer [emits a full list of native Media Events that you can listen to](https://github.com/SevenOutman/vue-aplayer/releases/tag/1.4.0). Among them you may want to use `timeupdate` to get informed that the current time of soundtrack is updated, then get the current playing time via `player.audio.currentTime`. Or you can simply watch `player.playStat.playedTime`, whose value keeps updating with `audio.currentTime`.

Btw this issue is a good place to share your project. You can leave your site link in comments.

**Comment by Shadomax** [comment]
okay thanks for your response but i want to know how to access it inside App.vue watch or computed.

**Comment by SevenOutman** [comment]
@Shadomax You may refer to [Vue docs on `ref`](https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements) to find out how you can access a child component instance.

**Comment by SevenOutman** [comment]
@Shadomax You may refer to [Vue docs on `ref`](https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements) to find out how you can access a child component instance.

**Comment by king-Senior** [comment]
我在项目中怎么监听pause 事件呢

**Comment by king-Senior** [comment]
How do I listen for pause events in my project

**Comment by Shadomax** [comment]
use @pause="pause()"

pause() is a function defined in your method object

**Comment by SevenOutman** [comment]
@Shadomax Check what I added to the beginning of this issue. Your comments will be deleted within 24 hours, or you can delete them yourself.

**Comment by SalocinDotTEN** [comment]
https://www.salocinten.info/#/audiorium

A combination of AWS Lambda, AWS S3, and scripting to produce a streaming random track audio player.

Edit: Temporarily not using it for now.

**Comment by admin-huang** [comment]
请问有播放多个音乐的demo吗？

**Comment by admin-huang** [comment]
Multiple music demonstrations

**Comment by EmmaNartey** [comment]
Want to attach my song data from my backend to vue-aplayer in my vue project but don't know how to go about it. If you can be of help, i will be most grateful.

**Comment by RaphaelDDL** [comment]
https://epicsevendb.com

A fan-made wiki / database for EpicSeven mobile game. I'm using vue-aplayer on Hero detail page, example: https://epicsevendb.com/hero/baiken

I'm pondering into changing to playlist. Visually I prefer multiple but since so many instances at same time made page a bit sluggish, I need to rethink.

**Comment by RuErJieFei** [comment]
为什么这么难用啊，一早上都没搞好

**Comment by mylove5** [comment]
我也想知道，为啥依赖都安装不上，说好的vue项目呢    npm run  dev    运行不了  0.0
